### PR TITLE
Added README.md file for better documentation.

### PR DESCRIPTION
**Pull request by born-2learn**
Removed the README.txt and added the README.md for a better documentation experience.

The README.md file can later prove useful once proper unit tests and dev cycles are setup for the website; and for many other purposes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
+# Scipy.org Website Source Code
+ 
+This repository contains the Sphinx source for the [SciPy website](http://www.scipy.org/).
 
 After cloning this repository, run
 
-    $ git submodule init
-    $ git submodule update
+`$ git submodule init`  
+`$ git submodule update`
```
Comment (mattip) [inline README.md]: Maybe make these into code blocks with three backticks (remove spacing added between for rendering):

``` 
` ` `
$git submodule init
$git submodule update
` ` `
```

Comment (born-2learn) [reply README.md]: Sure. Done with it.

**Comment by rgommers** [comment]
Merged, thanks @born-2learn 

> The README.md file can later prove useful once proper unit tests and dev cycles are setup for the website; and for many other purposes.

We don't plan to do much with this Sphinx-based site, at some point we'll replace it completely with a more modern-looking site, probably Hugo-based.

**Comment by born-2learn** [comment]
You're welcome @rgommers .
Oh, that's nice! The HUGO framework very user-friendly.

Let me know if there's anything to be done related to scipy.

**Pull request merged**